src/index.ts

```diff
@@ -1,2 +1,3 @@
-export { jsonToSchema } from "./schema-generator";
+export { jsonToSchema, addRule } from "./schema-generator";
```

Comment:
i think in the `index.ts` of `./schema-generator` it should export these 2 instead of `*` since `_testExports` is for testing

Reply:
ok have updated it